cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -420,6 +430,28 @@ traceD tr meta msg d = traceNamedObject tr (meta, LogValue msg (PureD d))
 traceI :: Integral i => Trace IO a -> LOMeta -> Text -> i -> IO ()
 traceI tr meta msg i = traceNamedObject tr (meta, LogValue msg (PureI (fromIntegral i)))
 
+sendEKGDirectInt :: Integral a => EKGDirect -> Text -> a -> IO ()
+sendEKGDirectInt ekgDirect name val = do
+ registeredMap <- readIORef (ekgGauges ekgDirect)
+ case SMap.lookup name registeredMap of
+   Just gauge -> Gauge.set gauge (fromIntegral val)
+   Nothing -> do
+     gauge <- EKG.getGauge name (ekgServer ekgDirect)
+     let registeredMap' = SMap.insert name gauge registeredMap
+     writeIORef (ekgGauges ekgDirect) registeredMap'
+     Gauge.set gauge (fromIntegral val)
+
+_sendEKGDirectDouble :: EKGDirect -> Text -> Double -> IO ()
+_sendEKGDirectDouble ekgDirect name val = do
```

Comment:
Why does this function start with an underscore?

Reply:
@newhoggy, this is currently unused, but will be immediately used by anyone who needs to post a `Double` metric.

We don't want then to be running around for recipes.. : -)